src/main/java/com/contrastsecurity/maven/plugin/VerifyContrastMavenPluginMojo.java

```diff
@@ -24,7 +24,19 @@ public void execute() throws MojoExecutionException {
 
         getLog().info("Checking for new vulnerabilities for appVersion [" + computedAppVersion + "]");
 
-        String applicationId = getApplicationId(contrast, appName);
+        String applicationId;
+        if (StringUtils.isNotBlank(appId)) {
+            applicationId = appId;
+
+            if (StringUtils.isNotBlank(appName)) {
+                getLog().info("Using 'appId' property; 'appName' property is ignored.");
+            }
+
+        } else if (StringUtils.isNotBlank(appName)) {
+            applicationId = getApplicationId(contrast, appName);
+        } else {
+            throw new MojoExecutionException("Either application id or name should be specified in plugin configuration.");
```

Comment:
Change `application id` and `name` to the actual property names as documented in the README

Reply:
👍